Wikipedia/assets/codemirror/codemirror-index.html

```diff
@@ -819,6 +819,23 @@
       focus: () => {
         editor.focus()
       },
+      focusWithoutScroll: () => {
+        const scrollY = window.scrollY
+        editor.focus()
```

Comment:
Might consider using this instead of `editor.focus()`...
<pre>
editor.display.input.focus({
     preventScroll: true
})
</pre>
... it bypasses a bunch of codemirror code that doesn't need to be called in this case (i think) and when webkit implements `preventScroll` we'd get it for free automatically.

If that sounds good may want a comment like this as a reminder for us later:

``
'preventScroll' is used here so when webkit starts supporting it we'll no longer have a jump. see:
https://html.spec.whatwg.org/multipage/interaction.html#dom-focusoptions-preventscroll
At that time we'll no longer need the `window.scrollTo` call below.
``

Reply:
Calling `focus` on `input` will always go through CodeMirror because of https://github.com/wikimedia/wikipedia-ios/blob/develop/Wikipedia/assets/codemirror/resources/lib/codemirror/lib/codemirror.js#L8853

It could take options and pass that to `this.div.focus()` but given that it doesn't do anything right now, I think it's not worth messing with `codemirror.js`. What do you think?

Reply:
ya prob not worth it 🧐